src/schema/sort.schema.ts

```diff
@@ -8,6 +8,11 @@ export interface Sort {
   order?: string;
 }
 
+export function isSort(object: any): object is Sort {
```

Comment:
First, I still do not get it. 
Second, if there is no way to understand this easily, should we ignore it?